src/app/my-donations/my-donations.component.scss

```diff
@@ -25,15 +25,19 @@ div.donation-container {
 }
 
 div.donation {
-  border-radius: 5px;
-  background: white;
-  margin-bottom: 10px;
-  width: 190px;
-  color: $colour-primary;
-  padding-left: 10px;
-  overflow-wrap: break-word;
-  flex-basis: 27%;
+  width: 225px;
```

Comment:
Not sure I get why we're going fixed width here, or why this particular width. Can we use `<biggive-grid/>` or possibly DIY `flex`, to stay closer to layouts that are already battle tested and consistently used in the rest of the apps?